### ISSUE TITLE
Update Codemirror

### ISSUE THREAD

**Issue by jhned** [WebDevStudios/WDS-Customizer-CSS#5]
I believe the version of Codemirror is causing false-positive linting errors for CSS variables, like this:

![image](https://user-images.githubusercontent.com/2520143/176764942-94aa17cf-0ba4-4c7c-86b4-5af09ec279bb.png)

![image](https://user-images.githubusercontent.com/2520143/176764973-f6a07962-5ca3-4710-a5ae-8386b0ee1176.png)

What's the best way to go about updating Codemirror so that this doesn't get flagged as an error?

**Comment by tw2113** [comment]
Given the age of the repo, I could easily believe codemirror needs to be updated. That said though, I'm not personally sure how to best provide that update with the bundled version we have here.

Just out of curiosity, do the errors go away if you declare the variable on `:root` ?

**Comment by jhned** [comment]
No, the issue persists.

And yeah, I get it... if it had been easy to see how to best update Codemirror, I would've attempted it myself-- thought I'd be better off asking someone closer to the project.

**Comment by tw2113** [comment]
For what it's worth, I'm not trying to be snarky or dismissive here, definitely acknowledging that we could use someone to review and update as needed, but I personally don't know how to get that done at the exact moment, and the only contributor to the repo has long not been an employee for WDS. I'll reach out internally with our current staff to look into this.

**Comment by jhned** [comment]
Yeah, no worries! Not trying to be passive-aggressive or whatever over here either. Thanks for the quick response!

**Comment by ksministries** [comment]
Wrong email and person
